gdal_subsetter/transform.py

```diff
@@ -111,21 +129,16 @@ def process_item(self, input_item: Item, source: HarmonySource) -> Item:
                     source, output_basename, input_filename, output_dir,
                     layernames
                 )
-            elif file_type == 'nc':
-                layernames, transformed_filename, process_msg = self.process_netcdf(
-                    source, output_basename, input_filename, output_dir,
-                    layernames
-                )
-
```

Comment:
This is important change number 1: removing the ability to cope with a netCDF4 input file.

`HarmonyAdapter.process_netcdf` and `HarmonyAdapter.nc2tiff` are able to be removed as they are only in support of a netCDF4 input (or netCDF4 input within a zip file).